Installing Mapbox Navigation SDK (beta) via `carthage` fails

Installing Mapbox Navigation SDK via `carthage` fails

**Mapbox Navigation SDK version:**
```
github "mapbox/mapbox-navigation-ios" "v0.17.0-beta.1"
```

**Carthage Version**: `carthage 0.29.0`
**macOS Version**: `macOS High Sierra, 10.13.4 (17E199)`
**Xcode version**:
```
$ xcodebuild -version
Xcode 9.3
Build version 9E145
```
**Swift version**:
```
$ swift --version
Apple Swift version 4.1 (swiftlang-902.0.48 clang-902.0.37.1)
Target: x86_64-apple-darwin17.5.0
```

**Cartfile**
```
github "mapbox/MapboxGeocoder.swift"
github "mapbox/mapbox-navigation-ios" "v0.17.0-beta.1"
```

```
$ carthage update --platform iOS
*** Cloning mapbox-navigation-ios
*** Cloning MapboxGeocoder.swift
*** Cloning mapbox-events-ios
*** Cloning mapbox-voice-swift
*** Downloading binary-only framework Mapbox-iOS-SDK at "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json"
*** Cloning MapboxDirections.swift
*** Checking out turf-swift at "v0.1.1"
*** Checking out MapboxDirections.swift at "v0.20.0"
*** Checking out trustkit at "1.5.3"
*** Checking out mapbox-navigation-ios at "v0.17.0-beta.1"
*** Checking out MapboxGeocoder.swift at "v0.8.1"
*** Checking out mapbox-voice-swift at "v0.0.1"
*** Checking out mapbox-events-ios at "v0.4.0"
*** xcodebuild output can be found in /var/folders/l4/kb9wf5dn5g31y8ctr6fj_f9c0000gn/T/carthage-xcodebuild.6zbWvV.log
[...]
*** Building scheme "MapboxCoreNavigation" in MapboxNavigation.xcodeproj
*** Building scheme "MapboxNavigation" in MapboxNavigation.xcodeproj
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -project /[...]/Carthage/Checkouts/mapbox-navigation-ios/MapboxNavigation.xcodeproj -scheme MapboxNavigation -configuration Release -derivedDataPath /[...]/Library/Caches/org.carthage.CarthageKit/DerivedData/9.3_9E145/mapbox-navigation-ios/v0.17.0-beta.1 -sdk iphonesimulator -destination platform=iOS\ Simulator,id=85A52CDB-9AEE-4292-A51A-E9580201B07A -destination-timeout 3 ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES build (launched in /[...]/Carthage/Checkouts/mapbox-navigation-ios)

This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/l4/kb9wf5dn5g31y8ctr6fj_f9c0000gn/T/carthage-xcodebuild.6zbWvV.log
```

**Carthage log file error message**
```
Ld /[...]/Library/Caches/org.carthage.CarthageKit/DerivedData/9.3_9E145/mapbox-navigation-ios/v0.17.0-beta.1/Build/Intermediates.noindex/MapboxNavigation.build/Release-iphonesimulator/MapboxNavigation.build/Objects-normal/i386/MapboxNavigation normal i386
    cd /[...]/Carthage/Checkouts/mapbox-navigation-ios
    export IPHONEOS_DEPLOYMENT_TARGET=9.0
    export PATH="/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/usr/bin:/Applications/Xcode.app/Contents/Developer/usr/bin:/[...]/.rvm/gems/ruby-2.5.1/bin:/[...]/.rvm/gems/ruby-2.5.1@global/bin:/[...]/.rvm/rubies/ruby-2.5.1/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/Applications/Server.app/Contents/ServerRoot/usr/bin:/Applications/Server.app/Contents/ServerRoot/usr/sbin:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/[...]/.rvm/bin"
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -arch i386 -dynamiclib -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator11.3.sdk -L/[...]/Library/Caches/org.carthage.CarthageKit/DerivedData/9.3_9E145/mapbox-navigation-ios/v0.17.0-beta.1/Build/Products/Release-iphonesimulator -F/[...]/Library/Caches/org.carthage.CarthageKit/DerivedData/9.3_9E145/mapbox-navigation-ios/v0.17.0-beta.1/Build/Products/Release-iphonesimulator -F/[...]/work/projects/COBI/001_cobi_app/001_development/COBI-iOS-App/Frameworks/Carthage/Checkouts/mapbox-navigation-ios/Carthage/Build/iOS -filelist /[...]/Library/Caches/org.carthage.CarthageKit/DerivedData/9.3_9E145/mapbox-navigation-ios/v0.17.0-beta.1/Build/Intermediates.noindex/MapboxNavigation.build/Release-iphonesimulator/MapboxNavigation.build/Objects-normal/i386/MapboxNavigation.LinkFileList -install_name @rpath/MapboxNavigation.framework/MapboxNavigation -Xlinker -rpath -Xlinker @executable_path/Frameworks -Xlinker -rpath -Xlinker @loader_path/Frameworks -mios-simulator-version-min=9.0 -dead_strip -Xlinker -object_path_lto -Xlinker /[...]/Library/Caches/org.carthage.CarthageKit/DerivedData/9.3_9E145/mapbox-navigation-ios/v0.17.0-beta.1/Build/Intermediates.noindex/MapboxNavigation.build/Release-iphonesimulator/MapboxNavigation.build/Objects-normal/i386/MapboxNavigation_lto.o -Xlinker -objc_abi_version -Xlinker 2 -fobjc-arc -fobjc-link-runtime -L/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/iphonesimulator -Xlinker -add_ast_path -Xlinker /Users/colins/Library/Caches/org.carthage.CarthageKit/DerivedData/9.3_9E145/mapbox-navigation-ios/v0.17.0-beta.1/Build/Intermediates.noindex/MapboxNavigation.build/Release-iphonesimulator/MapboxNavigation.build/Objects-normal/i386/MapboxNavigation.swiftmodule -framework Mapbox -framework Turf -compatibility_version 1 -current_version 25 -Xlinker -dependency_info -Xlinker /[...]/Library/Caches/org.carthage.CarthageKit/DerivedData/9.3_9E145/mapbox-navigation-ios/v0.17.0-beta.1/Build/Intermediates.noindex/MapboxNavigation.build/Release-iphonesimulator/MapboxNavigation.build/Objects-normal/i386/MapboxNavigation_dependency_info.dat -o /Users/colins/Library/Caches/org.carthage.CarthageKit/DerivedData/9.3_9E145/mapbox-navigation-ios/v0.17.0-beta.1/Build/Intermediates.noindex/MapboxNavigation.build/Release-iphonesimulator/MapboxNavigation.build/Objects-normal/i386/MapboxNavigation
ld: warning: ignoring file /[...]/Carthage/Checkouts/mapbox-navigation-ios/Carthage/Build/iOS/Mapbox.framework/Mapbox, missing required architecture i386 in file /[...]/Carthage/Checkouts/mapbox-navigation-ios/Carthage/Build/iOS/Mapbox.framework/Mapbox (3 slices)
Undefined symbols for architecture i386:
  "_OBJC_CLASS_$_MGLMultiPolylineFeature", referenced from:
      objc-class-ref in RouteMapViewController.o
  "_OBJC_CLASS_$_MGLAccountManager", referenced from:
      objc-class-ref in RouteMapViewController.o
  "_OBJC_CLASS_$_MGLPointAnnotation", referenced from:
      objc-class-ref in NavigationMapView.o
[...]
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -6,6 +6,7 @@
 
 * Upgraded to the [Mapbox Maps SDK for iOS v4.0.0](https://github.com/mapbox/mapbox-gl-native/releases/tag/ios-v4.0.0). If you have customized the route map’s appearance, you may need to migrate your code to use expressions instead of style functions. ([#1076](https://github.com/mapbox/mapbox-navigation-ios/pull/1076))
 * Added a Korean localization. ([#1346](https://github.com/mapbox/mapbox-navigation-ios/pull/1346))
+* Removed support for 32-bit simulators. [#1394](https://github.com/mapbox/mapbox-navigation-ios/pull/1394)
 
 ### User interface
 

--- a/MapboxNavigation.xcodeproj/project.pbxproj
+++ b/MapboxNavigation.xcodeproj/project.pbxproj
@@ -2606,6 +2606,7 @@
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
 				SWIFT_VERSION = 4.0;
 				TARGETED_DEVICE_FAMILY = "1,2";
+				"VALID_ARCHS[sdk=iphonesimulator*]" = x86_64;
 				VERSIONING_SYSTEM = "apple-generic";
 				VERSION_INFO_PREFIX = "";
 			};
@@ -2663,6 +2664,7 @@
 				SWIFT_VERSION = 4.0;
 				TARGETED_DEVICE_FAMILY = "1,2";
 				VALIDATE_PRODUCT = YES;
+				"VALID_ARCHS[sdk=iphonesimulator*]" = x86_64;
 				VERSIONING_SYSTEM = "apple-generic";
 				VERSION_INFO_PREFIX = "";
 			};

